ketcher-autotests/tests/User-Interface/Floating-windows/floating-windows.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { Page, test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  selectTopPanelButton,
+  waitForPageInit,
+  TopPanelButton,
+  readFileContents,
+  openFile,
+  pressButton,
+} from '@utils';
+
+async function openFileViaClipboard(filename: string, page: Page) {
+  const fileContent = await readFileContents(filename);
+  await page.getByText('Paste from clipboard').click();
+  await page.getByRole('dialog').getByRole('textbox').fill(fileContent);
+}
+
+async function returnText(page: Page) {
+  await page.getByTestId('openStructureModal').getByRole('textbox').click();
+  await page.keyboard.press('Home');
+  await page.keyboard.press('Enter');
+  await page.keyboard.press('Enter');
```

Comment:
What does this function do? I’m especially interested in the last three lines.

Reply:
Checking the possibility of editing pasted text  - step in the test "User has an ability to edit data in window".